server/text_generation_server/server.py

```diff
@@ -147,7 +174,7 @@ def serve(
     uds_path: Path,
 ):
     # Remove default handler
-    logger.remove()
+    #logger.remove()
```

Comment:
Yes, I want to remove this line. Because it causes the logger cannot print the logs in screen. In latest tgi, there lines are removed.  https://github.com/huggingface/text-generation-inference/blob/8dcc7d3f6ba5dacef1405a2cd1bb8e7b7531070f/server/text_generation_server/server.py#L207

Reply:
Yeah, my point was more about deleting this line for good, but it's a nit anyway :grin:

Reply:
No problem. When I upgrade the tgi-gaudi to v2.2.0, I will remove these lines.